### PR TITLE
add logic and test for nil tags on metrics points

### LINKED ISSUE #5: Crashing when event has no metrics attribute

### PR DESCRIPTION
**Pull request by jspaleta**
Closes #5

### CODE REVIEW

**Review by echlebek — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -257,32 +257,38 @@ func renderTemplates(event *corev2.Event) error {
...
-		case ts < 13:
-			// assume timestamp is milliseconds
-		case ts < 16:
-			// assume timestamp is microseconds
-			timestamp = (timestamp * 1000) / int64(time.Millisecond)
-		default:
-			// assume timestamp is nanoseconds
-			timestamp = timestamp / int64(time.Millisecond)
+	if event.Metrics != nil {
+		if event.Metrics.Points != nil {
```
[the remark this answers is not in the document]
Comment (jspaleta) [reply main.go]: yeah i was being over protective there. I'll run it back
Removed in new commit

File: main.go [hunk 2]
```diff
@@ -257,32 +257,38 @@ func renderTemplates(event *corev2.Event) error {
...
+			for _, point := range event.Metrics.Points {
+				tags := ""
+				if point.Tags != nil {
```
[the remark this answers is not in the document]
Comment (jspaleta) [reply main.go]: Removed in new commit
